src/servermap.cpp

```diff
@@ -639,8 +639,7 @@ void ServerMap::loadBlock(std::string *blob, v3s16 p3d, MapSector *sector, bool
 	try {
 		std::istringstream is(*blob, std::ios_base::binary);
 
-		u8 version = SER_FMT_VER_INVALID;
-		is.read((char*)&version, 1);
+		u8 version = readU8(is);
```

Comment:
So does EOF also result in the `is.fail()` condition being triggered?

Reply:
No idea, do you know?

Reply:
Alright then.